<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>palert</class>
 <widget class="QDialog" name="palert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>231</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Application:/usr/bin/firefox</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>221</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Protocol: TCP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>191</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Source: 10.0.0.2:5413</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>461</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination: 12.0.1.1:442(firefox.settings.services.mozilla.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Container:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>UID: 1000</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ALL Destination Addresses</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxADP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>All Destination Ports</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxASP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>All Source Ports</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxASA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>All Source Addresses</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>All Protocols</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAUID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>All UIDs</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Persistant</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>palert</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>palert</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
